index.js

```diff
@@ -674,22 +674,23 @@ class BitbucketScm extends Scm {
     _getCheckoutCommand(config) {
```

Comment:
Can you add jsdoc for `config.commitBranch`?

Reply:
@tkyi I added the comment to jsdoc. Thank you!